src/field_theory/normal.lean

```diff
@@ -25,7 +25,7 @@ noncomputable theory
 open_locale classical
 open polynomial is_scalar_tower
 
-universes u v
+universes u v w
```

Comment:
Is there a reason for making the universes explicit? Usually `Type*` just works, and does the right thing.

Reply:
Not that I know of. I was just following the prior convention in this file.

Reply:
Would you like me to change the variables to `Type*`?

Reply:
@tb65536 yeah, `Type*` used to have some hiccups, but nowadays it is the preferred option. So please convert to `Type*`, and after that, feel free to merge.

Thanks :tada: 

bors d+